Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
@DomT4 Is overwriting everything else a problem?

Ideally, we use the `install` target of the makefile because the upstream developers know what needs to be installed where. If we write our own install step, we open the door for a maintenance issue in the future when the upstream devs change the install step and we don't.

Running `make install` twice will do the job correctly, even when the upstream devs change things. The only downside is that it does more work than is strictly necessary.